examples/3d/3d_custom_image.rs

```diff
@@ -0,0 +1,186 @@
+    // so the UV coords will make more sense (note (0.0, 0.0) = Top-Left in UV mapping).
+    #[rustfmt::skip]
+    cube_mesh.insert_attribute(
+        Mesh::ATTRIBUTE_UV_0,
+        vec![
+            // Assigning the UV coords for the top side.
+            [0.5, 0.0], [0.0, 0.0], [0.0, 0.5], [0.5, 0.5],
+            // Assigning the UV coords for the bottom side.
+            [0.5, 0.5], [0.5, 0.0], [1.0, 0.0], [1.0, 0.5],
+            // Assigning the UV coords for the forward side.
+            [0.5, 1.0], [0.5, 0.5], [1.0, 0.5], [1.0, 1.0],
+            // Assigning the UV coords for the back side (same as forward because they have the
+            // same texture)
+            [0.5, 1.0], [0.5, 0.5], [1.0, 0.5], [1.0, 1.0],
+            // Assigning the UV coords for the right side.
+            [0.0, 1.0], [0.0, 0.5], [0.5, 0.5], [0.5, 1.0],
+            // Assigning the UV coords for the left side (same as right).
+            [0.0, 1.0], [0.0, 0.5], [0.5, 0.5], [0.5, 1.0],
+        ],
+    );
+
+    // When it comes to smooth, and simple meshes, normals are as simple as the direction of the flat surface
+    // Assign normals to allow for correct lighting calculations.
+    #[rustfmt::skip]
+    cube_mesh.insert_attribute(
+        Mesh::ATTRIBUTE_NORMAL,
+        vec![
+            // Normals for the top side (towards +y)
+            [0.0, 1.0, 0.0],
+            [0.0, 1.0, 0.0],
+            [0.0, 1.0, 0.0],
+            [0.0, 1.0, 0.0],
+            // Normals for the bottom side (towards -y)
+            [0.0, -1.0, 0.0],
+            [0.0, -1.0, 0.0],
+            [0.0, -1.0, 0.0],
+            [0.0, -1.0, 0.0],
+            // Normals for the forward side (towards +x)
+            [1.0, 0.0, 0.0],
+            [1.0, 0.0, 0.0],
+            [1.0, 0.0, 0.0],
+            [1.0, 0.0, 0.0],
+            // Normals for the back side (towards -x)
+            [-1.0, 0.0, 0.0],
+            [-1.0, 0.0, 0.0],
+            [-1.0, 0.0, 0.0],
+            [-1.0, 0.0, 0.0],
+            // Normals for the right side (towards +z)
+            [0.0, 0.0, 1.0],
+            [0.0, 0.0, 1.0],
+            [0.0, 0.0, 1.0],
+            [0.0, 0.0, 1.0],
+            // Normals for the left side (towards -z)
+            [0.0, 0.0, -1.0],
+            [0.0, 0.0, -1.0],
+            [0.0, 0.0, -1.0],
+            [0.0, 0.0, -1.0],
+        ],
+    );
+
+    // Create the triangles out of the 24 vertices we created.
+    // To construct a square, we need 2 triangles, therfore 12 traingles in total.
+    // To construct a triangle, we need the indices of 3 of its defined vertices, adding them one
+    // by one, in a counter-clockwise order (relative to the position of the viewer, the order
+    // should appear counter-clockwise from the front of the triangle). Read more about how to correctly build a mesh manually
+    // in the Bevy documentation, further examples and the implementation of the built-in shapes.
```

Comment:
Which documentation? I suspect you mean `Mesh`? Or maybe `StandardMaterial::cull_mode` for the vertex winding order?

This should be precise, otherwise it can be interpreted as talking down to the reader. (eg: "lol reader doesn't even know what docs they need to read")